### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by muigukenneth** [alex-townsend/SwipeOpenItemTouchHelper#10]
Hi guys,i get a npe when i try swiping to close an open item while helper.setCloseOnAction(true) .Check out the exception below.

FATAL EXCEPTION: main

```
                                                                 java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.ViewParent android.view.View.getParent()' on a null object reference
                                                                     at atownsend.swipeopenhelper.SwipeOpenItemTouchHelper.select(SwipeOpenItemTouchHelper.java:503)
                                                                     at atownsend.swipeopenhelper.SwipeOpenItemTouchHelper.checkSelectForSwipe(SwipeOpenItemTouchHelper.java:820)
                                                                     at atownsend.swipeopenhelper.SwipeOpenItemTouchHelper.access$700(SwipeOpenItemTouchHelper.java:51)
                                                                     at atownsend.swipeopenhelper.SwipeOpenItemTouchHelper$2.onInterceptTouchEvent(SwipeOpenItemTouchHelper.java:251)
                                                                     at android.support.v7.widget.RecyclerView.dispatchOnItemTouchIntercept(RecyclerView.java:2231)
                                                                     at android.support.v7.widget.RecyclerView.onInterceptTouchEvent(RecyclerView.java:2277)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2285)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2684)
                                                                     at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
                                                                     at com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2552)
                                                                     at com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1767)
                                                                     at android.app.Activity.dispatchTouchEvent(Activity.java:2840)
                                                                     at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:60)
                                                                     at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:60)
                                                                     at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2513)
                                                                     at android.view.View.dispatchPointerEvent(View.java:9125)
                                                                     at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4632)
                                                                     at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4470)
                                                                     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4020)
                                                                     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4073)
                                                                     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4039)
                                                                     at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4153)
                                                                     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4047)
                                                                     at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4210)
                                                                     at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4020)
                                                                     at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4073)
                                                                     at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4039)
                                                                     at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4047)
                                                                    at android.view.View
```

**Comment by alex-townsend** [comment]
Could you post your view holder code and layout? Also, are you using the latest version (1.0.0)?

**Comment by alex-townsend** [comment]
Nevermind, I replicated it. I'll have a fix in for this soon.

**Comment by alex-townsend** [comment]
Fixed with #11

**Comment by alex-townsend** [comment]
Should be fixed in the new 1.0.1 release. Please update and let me know if it fixed it for you.

**Comment by muigukenneth** [comment]
Thanks  alex-townsend ,i can confirm the the issue is now fixed on 1.0.1 release.
